runtime/src/error.rs

```diff
@@ -43,6 +43,8 @@ pub enum Error {
     KeyringAccountParsingError,
     #[error("Storage item not found")]
     StorageItemNotFound,
+    #[error("Client does not support node runtime")]
+    InvalidRuntimeVersion,
```

Comment:
nitpick: maybe it would be nice to include the expected and actual runtime versions in the error?